docs/README.md

```diff
@@ -1,30 +1,34 @@
-This guide will demonstrate how to regenerate the plots used in the documentation here.
-The requirements are
-1) Have docker installed
-2) Download the [PlotData] zip file containing the scenarios used
+### Requirements
+This guide will demonstrate how to regenerate the plots used in the
+documentation here. The only requirement is that docker is installed. The
+scenarios used will be downloaded from blob storage, which may take a while but
+should happen automatically.
 
-Next, extract the data to your home directory, or change the expected location in the
-docker-compose.yml file.
-
-
-Start the container, which will provide a notebook environment for running the code
-snippets.
+### Usage
+First, go to this directory, then start the container, which will provide a
+notebook environment for running the code snippets.
 ```
+cd docs
 docker compose up
 ```
 
-After copying the notebook url from the output, start the `run_snippets.ipynb` notebook
-and run through it. For the most part, no interaction is required. The exception is any
-bokeh plots which must be manually saved as png (currently only the powerflow snapshot). 
+Access the notebook in your browser using the localhost url given by jupyter.
+NOTE: if the host port is already in use by another container, it can be
+changed to one that is available (e.g. 10001). In this case, you'll need to
+change the port in the url to match, since the one given by jupyter is the port
+inside the container.
 
-One may notice that the first part of the notebook contains code to generate the cells
-used subsequently. This is due to (as far as I know) lack of support in jupyterlab for
-programmatically adding new cells, so as a result, any changes to the configuration may
-require a developer to rerun this part and copy/paste the output as needed. In most
-cases, one can simply run it with no changes, since the notebook is pre-populated.
+Now simply run the notebook to regenerate the plots. For the most part, no
+interaction is required. The exception is any bokeh plots which must be
+manually saved as png (currently only the powerflow snapshot). 
```

Comment:
Should we remove the parenthesis that mentions the PF snapshot since it has been removed from the notebook?

Reply:
We still have a cell to generate the plot, I only cleared the output content. So a user would still need to click save on the plot after running it.

Reply:
Sounds good